server/src/port/persistence/repository/repository.config.service.ts

```diff
@@ -0,0 +1,25 @@
+import {InjectConfig} from 'nestjs-config';
+import {Injectable} from '@nestjs/common';
+import {TypeOrmModuleOptions, TypeOrmOptionsFactory} from '@nestjs/typeorm';
+
+@Injectable()
+export class RepositoryConfigService implements TypeOrmOptionsFactory {
```

Comment:
If you implement `TypeOrmOptionsFactory`, should you name `RepositoryConfigFactory`?

Why does `RepositoryConfigService` implement `TypeOrmOptionsFactory`?

Reply:
Because I think name repositoryconfihservice is better

Reply:
it make sense ur comment but I think name repositoryconfihservice is better
1. We are using "service" at all the other place. i used service for consistency. 
2. other references are using name "service" at the same usage like `ConfigService`.

Reply:
Oh, I see!

I understand your mind. Thank you :)